data/competencies.yml

```diff
@@ -953,3 +953,236 @@
   level: senior2
   area: leadership
   domain: null
+
+# Tech lead, Technical
+
+- id: b6787605-ffa6-4243-a29f-2de90eba481a
+  summary: Works with the relevant Principal to develop and maintain a technical vision for the product, project or team
```

Comment:
**suggestion:** I'd be interested in some more examples of what format a "technical vision" could take, especially for a team that doesn't maintain a system (e.g. an experiment-focused team).

Reply:
>  I would roughly (and apologies for the inevitable poor description) describe work there more as "experimentation as a way of life" rather than a discovery team's "experimentation as a means to an end".

+1.

> Then the second question is "what physical format might that technical vision take?"

I didn't phrase my question very well. I was more interested in the contents of the vision, e.g. a roadmap of technical milestones, a one sentence vision statement. But I think the answer to that probably depends on the team.

> The third quetsion is "should the examples in this competencies.yml document be updated to be more meaningful to experiment-focused teams?" For this I think the answer is "no", but I'm happy for you to explain why I'm wrong.

I agree with you. I think what would help is examples of technical visions, ideally drawn from teams at the FT. I have never created one of these, or seen the technical vision of other teams I have worked in.

> A technical vision there might include something about building a leaner and more efficient way of running experiments. This might involve a combination technological and process improvements. Or perhaps its technical vision is more about being able to easily spin off successes[*] - to have a good way of packaging up a successful experiment so that it can be passed over to a build-phase team quickly and easily.

These are great suggestions, thank you.

Reply:
> I think what would help is examples of technical visions, ideally drawn from teams at the FT. I have never created one of these, or seen the technical vision of other teams I have worked in.

This sounds like a good opportunity for a discussion, a talk, a group workshop, or something similar. I will take that idea away and discuss it with others.